lib/vanagon/project.rb

```diff
@@ -302,7 +304,7 @@ def generate_package
     #
     # @param workdir [String] workdir to put the packaging files into
     def generate_packaging_artifacts(workdir)
-      @platform.generate_packaging_artifacts(workdir, @name, binding)
+      @platform.generate_packaging_artifacts(workdir, self, binding)
```

Comment:
You don't have to change this. You can access `@root_dir` with `@root_dir` in the `generate_packaging_artifacts` method.